CHANGELOG.md

```diff
@@ -22,10 +22,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   (defaults to `true`)
   - `DOCKER_VERNEMQ_ASTARTE_VMQ_PLUGIN__CASSANDRA__SSL_CUSTOM_SNI`
   - `DOCKER_VERNEMQ_ASTARTE_VMQ_PLUGIN__CASSANDRA__SSL_CA_FILE`
+  - `DOCKER_VERNEMQ_ASTARTE_VMQ_PLUGIN__ASTARTE_INSTANCE_ID`
+  (defaults to `default`)
```

Comment:
Move this to the log line to the one added in this PR. Also, the default is now `""`